src/api/internal.rs

```diff
@@ -186,6 +186,14 @@ impl InterConfig {
   pub(crate) fn keyframe_lookahead_distance(&self) -> u64 {
     cmp::max(1, self.max_reordering_latency()) + 1
   }
+
+  pub(crate) fn inter_refs_per_frame(&self) -> usize {
```

Comment:
Please name this function something else so it's not confused with the INTER_REFS_PER_FRAME constant.

Reply:
Maybe a better way to do this would be to return a list of reference frames that are used?